chall02/spentti.c

```diff
@@ -0,0 +1,32 @@
+#include <stdlib.h>
+
+char	*ft_itoa_hex(unsigned long long n, char *str, int i)
```

Comment:
This function could be written simpler, but it's good you came up with your own solution/converter